links/api/links.api

```diff
@@ -27,13 +27,13 @@ public final class com/splendo/kaluga/links/LinksManagerKt {
 }
 
 public abstract interface class com/splendo/kaluga/links/handler/LinksHandler {
-	public abstract fun extractQueryAsList (Ljava/lang/String;)Ljava/util/List;
+	public abstract fun extractQueryAsMap (Ljava/lang/String;)Ljava/util/Map;
```

Comment:
List<Any> --> Map<String, List<String>>, where key is a parameter name and value - a list of values (one parameter could have several values in the same url)